pkg/services/annotations/annotationsimpl/annotations.go

```diff
@@ -28,26 +30,42 @@ func ProvideService(
 ) *RepositoryImpl {
 	l := log.New("annotations")
 
+	xormStore := NewXormStore(cfg, log.New("annotations.sql"), db, tagService)
+
+	l.Debug("Using xorm write store")
```

Comment:
do we need this? Probably change it to "Initializing annotation repository" or even delete it